task1/FileInnerCounter.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+
+namespace task1
+{
+    public class FileInnerCounter
+    {
+        private const string path = "C:\\Temp\\UserText.txt";
+
+        public void Execute()
+        {
+            string text = File.ReadAllText(path);
```

Comment:
Since const is static member, we should call it with class name:
FileInnerCounter.Path